src/lib/download/downloadFiles.ts

```diff
@@ -70,7 +70,7 @@ export async function downloadFileAttachment(
 
 export async function downloadFileWithXhr(
   url: string,
-  extension?: string,
+  fileName?: string,
```

Comment:
It's not strictly the filename as it is appended to a timestamp? Maybe something like `filenameSuffix`?

Reply:
Changed the name